pmg/views.py

```diff
@@ -76,6 +76,31 @@ def classify_attachments(files):
 
     return audio, related
 
+@app.context_processor
+def inject_user_following():
+    default_meetings = load_from_api('v2/committee-meetings/', fields=['id','title','date','committee_id'])['results'][:10]
+    default_committees = load_from_api('v2/committees', return_everything=True, fields=['id','name','date'])['results'][:20]
+
+    if current_user.is_authenticated():
+        # Append user-followed committees if logged in
+        user_following = current_user.following
+        recent_meetings = []
+        cte_meetings = []
+
+        for committee in user_following:
```

Comment:
Some users follow 100+ committees. This is going to be too slow. We should use the same technique is in `user_followed_committee_meetings()` -- perhaps move that into a helper method on the user object itself, such as `recent_meetings_for_followed_committees`.